class.jetpack.php

```diff
@@ -2690,7 +2690,9 @@ public static function activate_default_modules( $min_version = false, $max_vers
 			}
 
 			// we'll override this later if the plugin can be included without fatal error
-			wp_safe_redirect( Jetpack::admin_url( 'page=jetpack' ) );
+			if ( $redirect ) {
```

Comment:
This is the one part of this PR i'm still unsure about. `activate_default_modules` can only safely be called from within wp-admin. Running this function via a remote request will be tricky. I'm still thinking on it.